CRM/Contribute/BAO/Contribution.php

```diff
@@ -1316,68 +1316,11 @@ public static function sortName($id) {
    * @return array
    */
   public static function annual($contactID) {
-    if (is_array($contactID)) {
-      $contactIDs = implode(',', $contactID);
+    if (!is_array($contactID)) {
+      $contactID = explode(',', $contactID);
```

Comment:
Should be $contactIds, surely? Or will it always only be one? Subsequent function's parameter says contactIds

Reply:
yep - although I think it practice it never is an array - changed